README.md

```diff
@@ -72,12 +86,29 @@ export const MyComponent = (): FC => {
 
   return (
     <div>
-      <Button variant={variant} onClick={doSomething} />
+      <Button variant={variant} onClick={doSomething}/>
     </div>
   );
 };
 ```
 
+you can also import a specific component directly
+
+```typescript jsx
+import {Badge} from "@casavo/habitat/badge"
+import {Button} from "@casavo/habitat/button"
+import {Checkbox} from "@casavo/habitat/checkbox"
+import {Switch} from "@casavo/habitat/switch"
+import { H1, H2, H3, H4, H5, H6, Body, Inline } from "@casavo/habitat/typography"
+```
+
+| Habitat Component | [RSC](https://www.plasmic.app/blog/how-react-server-components-work) |
```

Comment:
curiosity:
this table could go out of date easily.. there is a way to made it dynamic? 
I'm imaging a table like the bundle size table, that is generated when we open a pr

Reply:
Sure, it can be done... but I'm not sure it's worth it...
Honestly, I don't see how our components could change quickly enough to justify automating this process.
I might have missed something 😄